clickhouse_driver/columns/ipcolumn.py

```diff
@@ -0,0 +1,64 @@
+from ipaddress import IPv4Address, IPv6Address, AddressValueError
+
+from .. import errors
+from ..util import compat
+from .exceptions import ColumnTypeMismatchException
+from .stringcolumn import ByteFixedString
+from .intcolumn import UInt32Column
+
+
+class IPv4(UInt32Column):
```

Comment:
Let's add "Column" suffix to class name: `class IPv4Column`

Reply:
Done